src/elements/Divider/Divider.tsx

```diff
@@ -26,22 +24,11 @@ export type DividerProps = {
 export default function Divider({
     style = DividerStyle.Solid,
     height = DividerHeight.Height50,
-    color = "#CCC",
+    color: borderTopColor = "#CCC",
```

Comment:
wouldn't it make sense to reuse an existing color in that case?

Reply:
atm i would leave it like that, since the devider isn't "designed" yet... so it could potentially change how we will use...